### ISSUE TITLE
Placenta: Fetal vascular malperfusion

### ISSUE THREAD

**Issue by janine-pappas** [obophenotype/human-phenotype-ontology#6578]
Preferred term label: **Fetal vascular malperfusion**
Synonyms **Fetal thrombotic vasculopathy**
Definition (free text, please give PubMed ID) **placental lesions indicating reduced or absent perfusion of the villous parenchyma by the fetus** 
PMID: **27223167**
https://pubmed.ncbi.nlm.nih.gov/27223167/
Parent term (use hpo.jax.org/app)   **HP:0100767 Abnormal Placental Morphology**  
Your nano-attribution (ORCID)**  **https://orcid.org/0000-0002-9579-3626**

**Comment by pnrobinson** [comment]
@janine-pappas   -- should this term really be placed underneath "placental membrane"? The cause of the lesion can "result from a number of conditions (eg, umbilical cord lesions, hypercoagulability, complications of fetal cardiac dysfunction, such as hypoxia, etc.)"
As far as I understand, the location of the observed abnormalities is not in the membranes themselves. --  Is the actual phenotypic obsservation "avascular villi"

**Comment by janine-pappas** [comment]
Hi Peter,  when I asked our Director of Placental Pathology, Rebecca Linn, she stated that if placental disc is not an option they should be placed under Abnormal placental morphology.

**Comment by pnrobinson** [comment]
@janine-pappas thanks, that sounds reasonable. We are in the process of updating the prenatal / perinatal part of the HPO and will be adding more detail to this section in 2021, so we will probably add terms for placental disc etc and move these items at that time.

**Comment by pnrobinson** [comment]
@janine-pappas Looking into this in a little more detail, it seems that the phrase "Fetal vascular malperfusion" is a grouping phrase to describe a variety of findings including Thrombosis, Avascular Villi, Intramural Fibrin Deposition,Villous Stromal-Vascular Karyorrhexis, and Stem Vessel Obliteration.
The HPO is intended to provide detailed terms for 'deep phenotyping', i.e., to describe the individual components to allow geonotype/phenotype correlation studies etc. I think it would be better to work on the detailed terms. It seems that the 'vascular malperfusion' is an inference about the pathophysiology but not really a phenotypic observation. I am going to keep this issue open -- we will be working on this area in a series of workshops in the next several months. Send me an email if you would like to join.

**Comment by pnrobinson** [comment]
added term after discussion  with prenatal group.
